alf/examples/atari.gin

```diff
@@ -1,5 +1,5 @@
 # environment config
-import tf_agents.environments.suite_gym
+import alf.environments.suite_gym
```

Comment:
perhaps it's better to `import alf.environments.suite_*` in `alf/environments/utils`  , so avoid to import suite_* for every gin configuration file.  

[L19-L21 modification ](https://github.com/HorizonRobotics/alf/commit/22142289d9c9d609c82fa655f1703864c6e50af7#diff-0d12de508ba2ef2d70e9ec2274a9c09cL19-L21) in  #211    make some   examples for super mario and social bot not 
 available

Reply:
> And [L19-L21 modification ](https://github.com/HorizonRobotics/alf/commit/22142289d9c9d609c82fa655f1703864c6e50af7#diff-0d12de508ba2ef2d70e9ec2274a9c09cL19-L21) in #211 make some examples for super mario and social bot not
> available

What exact modifications are you referring to?

Reply:
If we import all suite_* in utils, then there is a circular import dependency which we should always avoid. I think it makes more sense to move some common utils like `ProcessPyEnvironment` and `UnwrappedEnvChecker` to `utils.py` and import suite_* in individual gin files.

Reply:
What I can do at most is to import suite_* in `utils.py` for suites that don't rely on `utils.py`: `suite_gym` and `suite_simple`.

Reply:
ok , i see